articles/docker.md

```diff
@@ -6,11 +6,11 @@ excerpt: The Synapse Docker registry provides a space for synapse users to store
 
 ## Synapse Docker Registry
 
-Docker containers wrap a piece of software in a complete filesystem that contains everything needed to run.  This can be extremely helpful as software can have many dependencies, so by installing all of them in a container, users can avoid going through the trouble of installing the software on their own computer.  These Docker images can then be stored and distributed on a Docker registry.  The Synapse Docker registry will allow users to create software on a per project basis which can be easily shared across synapse. To learn more about [Docker](https://www.docker.com/products/overview) and [Docker registry](https://www.docker.com/products/docker-registry)
+Docker is a tool for creating, running and managing lightweight virtual machines, represented as versioned 'repositories.'  These virtual machines make it possible to distribute executable environments with all of the dependencies that can easily be run by others.  A Docker 'registry' is a collection of these repositories and there are a number of open registries on the web. Synapse hosts a private registry, freely available to our users. Synapse users interact with the Synapse Docker registry using the standard Docker client. As with Files and Table, Repositories are organized by project and inherit the access controls from the project.
 
 
 ### Creating a new Docker image
-Lets begin by creating a custom docker image.  Users can choose to either modify an existing docker image or build a docker image from a Dockerfile.  Docker images must be tagged with 'docker.synapse.org/synapseProjectId/myreponame' to allow images to be saved. 
+Users can choose to either modify an existing docker image or build a docker image from a Dockerfile.  Docker images must be tagged with 'docker.synapse.org/synapseProjectId/myreponame' to allow images to be saved.  In this example we will pull down an existing Docker image and add it to the Synapse project, 'syn12345'. 
```

Comment:
'docker' is a proper noun (per Docker's documentation) - should be capitalized in all places in this document.